hack/env.sh

```diff
@@ -9,3 +9,4 @@ export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE=${SRIOV_NETWORK_CONFIG_DAEMON_IMAGE:-qu
 export RELEASE_VERSION=0.0.1-snapshot
 export OPERATOR_NAME=sriov-network-operator
 export RESOURCE_PREFIX=openshift.io
+export ENABLE_ADMISSION_CONTROLLER='true'
```

Comment:
Doesn't this enable the resources mutating webhook (and only when running locally) ?
What is its relation with these new webhooks?

Reply:
With https://github.com/openshift/sriov-network-operator/pull/94, Operaoter controls SRIOV admission controller (resource injector) via `SriovOperatorConfig` CRD (spec.EnableInjector). Env var (ENABLE_ADMISSION_CONTROLLER) in this PR is going to enable/disable for the Operator's own admission controller which is used to mutate/validate Operator's CRD.

Reply:
Oh, I see.